src/plays/states/States.jsx

```diff
@@ -3,10 +3,9 @@ import { getPlayById } from "meta/play-meta-util";
 import PlayHeader from "common/playlists/PlayHeader";
 import "./states.css";
 
-function States(props) {
-  // Do not remove the below lines.
-  // The following code is to fetch the current play from the URL
-  const { id } = props;
+function States() {
+  const location = useLocation();
```

Comment:
We are not using useLocation any more. The code was refactored to fix the deeplinking bug. This changes reverts the bug fixing.

Reply:
Okay, I understood what mistake I did. I think I have resolved the conflicts.